### DISCUSSION TITLE
6-Dof tracking

### DISCUSSION THREAD

**Discussion by DaveyDave123** [introlab/rtabmap#1160] [General]
Hi All,
I hope you could give me a few pointers please. I'm looking for a 6 DoF tracking function, possibly in association with a stereo camera presumably using visual odometry techniques - I don't have any particular requirement for mapping. I've previously considered the Intel Realsense T265 sensor as a good all in one solution, but now this is obsolete. So, my question is,  would RTABMap be a good way of achieving 6 DoF tracking if I combined it with a stereo camera pair, or is that not really what RTabMap is used for?
Thanks in advance,
Dave

**Comment by matlabbe** [comment]
You can try RTAB-Map Windows binaries [here](https://github.com/introlab/rtabmap/releases/tag/0.21.0), which support some common stereo cameras out-of-the-box (see section drivers). If you need the output pose in real-time, I would recommend to use the ros package [rtabmap_ros](https://github.com/introlab/rtabmap_ros) instead. If you are not a ros user, it is also possible to use RTAB-Map as a library in c++, example [here](https://github.com/introlab/rtabmap/wiki/Cplusplus-RGBD-Mapping) (you can use only `Odometry` class if you care only about odometry).
